_posts/2020-01-23-scatter-gather-pt2.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title: Scatter-Gather in NServiceBus, part 2
+---
+
+In the [previous blog post](/2019/09/25/scatter-gather-pt1.html) we concluded that sagas struggle to deal with scenarios where a multiple messages need to update the same saga instance. Depending on the persistence option, this can manifest differently in practice:
```

Comment:
```suggestion
In the [previous blog post](/2019/09/25/scatter-gather-pt1.html) we concluded that sagas struggle to deal with scenarios where multiple messages need to update the same saga instance. Depending on the persistence option, this can manifest itself differently in practice:
```